packages/happy-dom/src/event/Event.ts

```diff
@@ -17,6 +18,7 @@ export default class Event {
 	public _propagationStopped = false;
 	public _target: IEventTarget = null;
 	public _currentTarget: IEventTarget = null;
+	public timeStamp: number = Date.now() - performance.timeOrigin;
```

Comment:
Yeah true, good catch! Not sure how I missed the `performance.now()` as it's describes exactly what we need:
> The returned value represents the time elapsed since the [time origin](https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp#the_time_origin).

😀